client/my-sites/stats/stats-subscribers-highlight-section/style.scss

```diff
@@ -11,7 +11,14 @@
 	// Override the default minimum width.
 	.highlight-card {
 		min-width: 240px;
+
+		// make it more compact for is-compact
+		&.is-compact {
+			min-width: 0;
```

Comment:
We might want to keep a reasonable min-width here, otherwise if the screen is too narrow, the texts would overlap on each other. What do you think?

<img width="173" alt="image" src="https://github.com/Automattic/wp-calypso/assets/1425433/fbc31b60-658b-4801-baab-f8185e72e5d4">

Reply:
yeah, that makes a lot of sense 😅  Thanks Jasper!

Reply:
Adding a minimum width will be even more beneficial when we add a chart.